src/App.tsx

```diff
@@ -1,7 +1,35 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string;
+};
+
+export class App extends Component <{}, State> {
+  state = {
+    pressedKey: 'Nothing was pressed yet',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.pressedKeyValue);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.pressedKeyValue);
+  }
+
+  pressedKeyValue = (event: KeyboardEvent) => {
```

Comment:
not a big mistake, but it would be better for function name specify the action, for example, `pressKey`, `handlePressKey` or  `handleKeyup` etc.